src/components/chip/chip.tsx

```diff
@@ -0,0 +1,279 @@
+import {
+    Component,
+    Element,
+    Event,
+    EventEmitter,
+    h,
+    Prop,
+} from '@stencil/core';
+import { Icon, Languages, Link } from '../../interface';
+import { getIconName } from '../icon/get-icon-props';
+import {
+    makeEnterClickable,
+    removeEnterClickable,
+} from '../../util/make-enter-clickable';
+import translate from '../../global/translations';
+import {
+    BACKSPACE,
+    BACKSPACE_KEY_CODE,
+    DELETE,
+    DELETE_KEY_CODE,
+} from '../../util/keycodes';
+
+/**
+ * Chips and buttons are both interactive elements in UI design,
+ * but they serve different purposes and are used in different contexts.
+ *
+ * :::warning
+ * Do not use the chip component, carelessly as an alternative for
+ * [`limel-button`](#/component/limel-button/) in the UI design!
+ * In short, buttons are used to trigger actions,
+ * while chips are used to represent entities or choices.
+ *
+ * **Buttons:**
+ * Buttons are used to trigger actions. They are typically used to
+ * submit forms, open dialogs, initiate a process, or perform any action
+ * that changes the state of the application.
+ * Buttons' labels usually contain action words, in other words, the labels is
+ * a _verb in imperative mood_ such as "Submit" or "Delete".
+ * Buttons are placed in areas where it's clear they will initiate
+ * an action when clicked.
+ *
+ * **Chips:**
+ * Chips are compact elements that represent an input, attribute, or action;
```

Comment:
I'd suggest that we probably don't mention "action" here. I understand the difference between "triggering an action" and "representing an action as a 'thing'", but I'm worried it's a little too confusing in this fairly compact context.

(I don't know, maybe it's not a problem, but it made me go "hang on, didn't he just say… ah, wait, no, I understand", and that makes me suspect that someone less familiar with the whole thing might be a lot more confused for a lot longer 😅)

Reply:
Now I'm _very_ confused 😅

![image](https://github.com/Lundalogik/lime-elements/assets/154725/af6970b3-1908-4abe-a9a6-1eef2633e6f0)

Reply:
me too 😅

Reply:
[`27a6aaa` (#2707)](https://github.com/Lundalogik/lime-elements/pull/2707/commits/27a6aaac38dfee901c07ec5782ac75eee6356247)

Reply:
Well… it became a little less obviously self-contradictory, but I don't think it really became any clearer…

Perhaps we can have a huddle, and you can explain to me what you want to get across, and then I can perhaps help formulate that in some way? 😊

Reply:
Ok I buy it. Added a new fixup c6b21dbf85a3a96d9635b3b2be198f37c44a86c1 but I can't really explain it more. These are like unwritten design rules. Maybe I'm one of the first people trying to write something meaningful about this topic 😅

Reply:
internally for our consultants and such, I have 0 hope that they even look at this docs. Sorry, I have -100 hope that they even care to read a second. Many seem to just go it and see if they like how it looks, then jut copy/paste the code into their project 😞

Reply:
> Ok I buy it. Added a new fixup [c6b21db](https://github.com/Lundalogik/lime-elements/commit/c6b21dbf85a3a96d9635b3b2be198f37c44a86c1) but I can't really explain it better. These are like unwritten design rules. Maybe I'm one of the first people trying to write something meaningful about this topic 😅

I think the updated explanation is great! 🙌

> Many seem to just go it and see if they like how it looks, then jut copy/paste the code into their project 😞

Mmm… yeah, I know 😞

"simulating network delay"… 😣